### DISCUSSION TITLE
Cloudflare and Two Rocket Servers

### DISCUSSION THREAD

**Discussion by tuckotb** [rwf2/Rocket#1861] [Questions]
I'm using two Rocket servers (one for frontend and one for api) and trying to use cloudflare with both. There both hosted on the same server, the first has port 80 and the second is on port 8443 but the second one gives a "SSL handshake failed" on Cloudflare. Changing ports did not help. Is there any way to overcome this error?

**Comment by jebrosen** [comment]
This looks more like a question about Cloudflare about Rocket. Please see Cloudflare's documentation or support channels to determine if you are supposed to configure either/both of your servers for HTTP or HTTPS, and/or if Cloudflare can terminate SSL, for your individual situation.

In case you do need to configure TLS for one of the Rocket servers, this is documented in Rocket's [Configuration guide](https://rocket.rs/v0.5-rc/guide/configuration/#tls).

**Reply by tuckotb** [reply]
I'll try TLS with the cert and key that Cloudflare gives us.
